ENIGMAsystem/SHELL/Widget_Systems/xlib/zenity.cpp

```diff
@@ -181,7 +174,7 @@ static inline void show_debug_message_helper(string errortext, MESSAGE_TYPE type
   if (strtod(str_result.c_str(), NULL) == 1) exit(0);
 }
 
-namespace enigma_user {
+namespace zenity {
```

Comment:
Do the same thing down here for Zenity, and all of this should work with no additional code to define new implementations.